crrExistingObjects.js

```diff
@@ -104,7 +104,7 @@ function _markObjectPending(bucket, key, versionId, storageClass,
         }, log, next),
         (mdRes, next) => {
             objMD = new ObjectMD(mdRes);
-            const mdBlob = objMD.getSerialized();
+            const mdBlob = objMD.getValue();
```

Comment:
The name `mdBlob` is confusing because it's not a blob anymore but a JS object, I suggest changing the variable name.

Reply:
Updated comment with improvements made.